src/components/enum-condition/enum-condition.tsx

```diff
@@ -151,15 +155,19 @@ export const EnumCondition = observer(
 
     return (
       <>
-        <AttributeHeader
-          chosenAttributes={selectedVariants.length}
-          allAttributes={enumVariants.length}
-          attrStatus={filterStore.selectedAttributeStatus!}
-        />
+        {isRefiner && (
```

Comment:
It's not good way to use `isRefiner` prop in this component.
<AttributeHeader /> should be in the middle column of filter refiner component